index.html

```diff
@@ -680,8 +738,13 @@ <h1 style="font-size: 48px;">My headline</h1>
 
 						---
 
+						### How is CSS applied?
+						There are a few rules that determine how styles are applied to your HTML. They all work together, and can combine in interesting and unexpected ways.
```

Comment:
I did a fair amount of reworking of this section, splitting it into the cascade, inheritance, and specificity. I wish there was a faster way to cover this, but I've never found one, and it is fairly important to help them be able to debug.